tasks/gitlab_config_files.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Place unicorn config 
+  template:
+    src: templates/unicorn.rb.j2
+    dest: "{{ gitlab_install_path | mandatory }}/config/unicorn.rb"
+  when: gitlab_unicorn_config_path is not defined
+
+- name: Copy unicorn config
+  copy: 
+    src: "{{ gitlab_unicorn_config_path }}"
+    dest: "{{ gitlab_install_path | mandatory }}/config/unicorn.rb"
+  when: gitlab_unicorn_config_path is defined
+
+- name: Copy the rack attack config
+  copy:
+    src: "{{ gitlab_rack_attack_config_path | default('files/rack_attack.rb')}}"
+    dest: "{{ gitlab_install_path | mandatory }}/config/rack_attack.rb"
+
+- name: Place resque config 
+  template:
+    src: templates/resque.yml.j2
+    dest: "{{ gitlab_install_path | mandatory }}/config/resque.yml"
+
+- name: Copy database config
+  template:
+    src: templates/database.yml.j2
+    dest: "{{ gitlab_install_path | mandatory }}/config/database.yml"
+    owner: git
+    mode: 0640
```

Comment:
Why specify permissions and owner here and nowhere else? Also, if give the file to root:git and use 640 as permission, the git user is unable to change permissions of the file which will increase security

Reply:
Of course, this won't work because you give all files to the git user in gitlab_permissions.yml...